src/main/java/BobCat/model/TaskList.java

```diff
@@ -0,0 +1,53 @@
+package BobCat.model;
+
+import BobCat.exception.InvalidOpsException;
+import BobCat.model.task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private final ArrayList<Task> storage;
```

Comment:
The variable name here should be in uppercase I believe.

Reply:
Can I clarify that the use of uppercase here is due to the "final" keyword? Was under the impression that constants in Java refer to static final fields, not just final